roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -0,0 +1,633 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# pylint: disable=line-too-long,invalid-name
+
+"""For details on this module see DOCUMENTATION (below)"""
+
+# router/registry cert grabbing
+import subprocess
+# etcd config file
+import ConfigParser
+# Expiration parsing
+import datetime
+# File path stuff
+import os
+# Config file parsing
+import yaml
+# Certificate loading
+import OpenSSL.crypto
+
+DOCUMENTATION = '''
```

Comment:
For the modules that we ship I wonder if we should set the `version_added` key to the version of origin/ocp that the module first appeared in. I guess for this case then we'd set it to either `1.3` or `3.3` depending on which you prefer to use.

Also. The doc generator is great. I've ran that on modules I've written before and it works great (it just took a little rejiggering).